manual/manual/cmds/comp.etex

```diff
@@ -100,6 +100,17 @@ phase can be run directly, as in:
 The produced file has the executable bit set, and it manages to launch
 the bytecode interpreter by itself.
 
+The compiler is able to emit some information on its internal stages.
+It can output ".cmt" files for the implementation of the compilation unit and
+".cmti" for signatures. This operation is activated by passing the key
+"-bin-annot" to the compiler (see the description of "-bin-annot" below).
+Each such file contains a typed abstract syntax tree (AST), that is produced
+during the type checking procedure. This tree contains complete information
+about the location and the specific type of each term in the source file.
+AST is partial if type checking was unsuccessful.
```

Comment:
It is better to not start with an abbreviation: `The AST is partial ...`.
Also if the tree is partial, the information is not complete. Maybe "all available" rather than "complete"?